requirements.txt

```diff
@@ -75,8 +75,8 @@ pip==24.0
 platformdirs==4.1.0
 psycopg2-binary==2.9.5
 pycparser==2.21
-pydantic==2.5.3
-pydantic_core==2.18.2
+#pydantic==2.5.3
+#pydantic_core==2.18.2
```

Comment:
Pydantic is used for the Structured Outputs logic, so these need to stay. I'd question whether the sim would actually run without them